.travis.yml

```diff
@@ -1,14 +1,15 @@
-language: c
-sudo: required
-dist: trusty
+language: minimal
 
 services:
   - docker
 
+env:
+  - DIST=fedora PKGMGR=dnf
+  - DIST=debian PKGMGR=aptcc
```

Comment:
Maybe BACKEND would be a better variable name here to match with -Dpackaging_backend?

Reply:
I suppose. I named it as such to make clear when glancing at the job overview page.